README.md

```diff
@@ -1,24 +1,203 @@
-# Angular Training Slides
+# Angular Training Materials
 
-Slides for the Angular 2+ training created with Reveal.js and using Markdown format
+This repository contains slides and other materials
```

Comment:
Are we doing manual line breaks for content?  I think we should probably be consistent on this